test/functional/pdo_sqlsrv/pdo_ae_azure_key_vault_common.php

```diff
@@ -0,0 +1,94 @@
+<?php
+require_once("MsCommon_mid-refactor.inc");
+require_once("MsSetup.inc");
+require_once('values.php');
+
+// Set up the columns and build the insert query. Each data type has an
+// AE-encrypted and a non-encrypted column side by side in the table.
+// If column encryption is not set in MsSetup.inc, this function simply
+// creates two non-encrypted columns side-by-side for each type.

```

Comment:
Not sure if I agree with your comment here because you have a specific connection string in your test, so it's not really affected by the flags in MsSetup?

Reply:
ColumnMeta does use the flags in MsSetup though, so setting $keystore and $dataEncrypted to encrypt in MsSetup will encrypt the _AE columns if the rest of the connection string enables encryption.